public/clinics.js

```diff
@@ -85,6 +85,53 @@ function createCompanyAddressSection(index, id, name, address) {
         addressInput.value = address;
     }
     section.appendChild(addressInput);
+    section.appendChild(document.createElement('br'));
+
+    const abnLabel = document.createElement('label');
+    abnLabel.textContent = 'ABN:';
+    section.appendChild(abnLabel);
+    section.appendChild(document.createElement('br'));
+
+    const abnInput = document.createElement('input');
+    abnInput.type = 'text';
+    abnInput.classList.add('abn');
+    abnInput.name = 'abn';
+    if (abn) {
+        abnInput.value = abn;
+    }
+    section.appendChild(abnInput);
+    section.appendChild(document.createElement('br'));
+
+    const postcodeLabel = document.createElement('label');
+    postcodeLabel.textContent = 'Postcode:';
```

Comment:
That is not what the spec says the line should say, please check your ticket and correct. The only difference to the spec is that is show two address lines, but we are only implementing 1, all other details need to be exact unless Dr Jim says otherwise.